config/webpack/plugin/TerraDevSite.js

```diff
@@ -34,6 +34,7 @@ class TerraDevSite {
   constructor({ env = {}, sites = [] } = {}) {
     // Validate sites.
     validate(sites);
+    this.inTest = env.inTest || false;
```

Comment:
Are we going to open this up to accept a process ENV so teams can run the site in the same way it's tested?

Reply:
They can, for example i use this as an npm script to test this out. "start-test": "tt-serve --env.inTest", I don't _think_ there is a need for this to also be an ENV 🤷‍♂